Canvas/Info.plist

```diff
@@ -30,5 +30,9 @@
 	<string>MainMenu</string>
 	<key>NSPrincipalClass</key>
 	<string>NSApplication</string>
+	<key>CFBundleDisplayName</key>
+	<string></string>
+	<key>SUFeedURL</key>
```

Comment:
Do we need this for Canvas?

Reply:
Yup, sorry about that, this will just be needed for Brooklyn I believe.